frontend/src/pages/HomePage.js

```diff
@@ -0,0 +1,16 @@
+import {Typography} from "@mui/material";
+import PortfolioItem from "../components/PortfolioItem";
+
+export default function HomePage({stocks}) {
+
+    return (
+        <div>
+            <Typography variant="h3">HomePage</Typography>
+            {stocks.map(stock => {
+                return <PortfolioItem stock={stock} key={stock.id}/>
```

Comment:
double return? it think you can write 
           {
                stocks.map(stock => <PortfolioItem stock={stock} key={stock.id}/>)
            }